osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -0,0 +1,268 @@
+                )
+                os.chdir(tmp_dir)
+                # use newest images
+                subprocess.check_output("docker-compose pull".split(' '))
+                # database needs to be ready
+                subprocess.check_output("docker-compose up -d db".split(' '))
+                # wait for the db to be up
+                subprocess.check_output("sleep 10".split(' '))
+
+                inform_user(
+                    extraction_order.orderer,
+                    messages.INFO,
+                    _('The GIS extraction of the order "%s" is has been started.') % extraction_order.id,
+                    email=False
+                )
+
+                bounding_geometry = extraction_order.excerpt.bounding_geometry
+                if type(bounding_geometry) == models.BBoxBoundingGeometry:
+                    bbox_args = '%(excerpt_south_border)s %(excerpt_west_border)s %(excerpt_north_border)s ' \
+                        '%(excerpt_east_border)s' % {
+                            'excerpt_north_border': bounding_geometry.north,
+                            'excerpt_west_border': bounding_geometry.west,
+                            'excerpt_south_border': bounding_geometry.south,
+                            'excerpt_east_border': bounding_geometry.east
+                        }
+
+                    if len(execution_configuration['formats']) > 0:
+                        subprocess.check_call(("docker-compose run bootstrap sh main-bootstrap.sh %s" %
+                                               bbox_args).split(' '))
+                        extraction_order.state = models.ExtractionOrderState.PROCESSING
+                        extraction_order.save()
+                        GisExcerptConverter.extract_excerpts(
+                            execution_configuration,
+                            extraction_order,
+                            bbox_args
+                        )
+
+                elif type(bounding_geometry) == models.OsmosisPolygonFilterBoundingGeometry:
+                    inform_user(
+                        extraction_order.orderer,
+                        messages.ERROR,
+                        _('GIS excerpt converter is not yet able to extract polygon excerpts.'),
+                        email=False
+                    )
+
+                else:
+                    inform_user(
+                        extraction_order.orderer,
+                        messages.ERROR,
+                        _('GIS excerpt converter is not yet able to extract excerpts of type %s.') %
+                        type(bounding_geometry).__name__,
+                        email=False
+                    )
+
+                subprocess.check_call("docker-compose stop --timeout 0".split(' '))
+                subprocess.check_call("docker-compose rm -vf".split(' '))
+
+                file_conversion_finished(extraction_order.orderer)
+            except:
```

Comment:
In production, don't show the trace to the user. Log it instead.

In development, consider raising the exception.